### DISCUSSION TITLE
Recommended settings for a 'CR10 Smart'?

### DISCUSSION THREAD

**Discussion by kriegtiger** [supermerill/SuperSlicer#1684] [Q&A]
Subject says it all - if someone is using SuperSlicer with a 'CR10 Smart' I would be in your debt if you could share your printer profile settings with me, I'm fairly new into 3d printing and building a profile from scratch is still pretty intimidating for some settings.

**Comment by supermerill** [comment]
You can use the creality cr-10 profile from the wizard.
As it has an auto levelling thing, you can change the z step from 0.04 to 0.005, but it's not mandatory.
